spec/aggregate_root_spec.rb

```diff
@@ -3,16 +3,18 @@
 class Order
   include RailsEventStore::AggregateRoot
 
+  attr_accessor :id, :status
```

Comment:
This will be described in a blog post which I plan to publish soon (need to finish sample code) - you should not check (or really care) about the aggregate internal state, the only thing you care about and what you should check is what events have been produced by calling aggregate's method (executing a command). The state of an aggregate is internal - it just an implementation detail, for some operations it could be no state representation at all - only new produced events).